src/ring_theory/polynomial/cyclotomic/analytic.lean

```diff
@@ -0,0 +1,108 @@
+/-
+Copyright (c) 2021 Eric Rodriguez. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Rodriguez
+-/
+
+import ring_theory.polynomial.cyclotomic.basic
+import tactic.by_contra
+import topology.algebra.polynomial
+import number_theory.padics.padic_norm
+import group_theory.specific_groups.real
+
+/-!
+# Evaluating cyclotomic polynomials
```

Comment:
Does it make sense to call this file `eval` or something similar, and move here results like `polynomial.eval_one_cyclotomic_prime`?

Reply:
That's a great name, I think, yes - I'll change it!